lib/OpenQA/Schema/Result/Bugs.pm

```diff
@@ -78,7 +78,7 @@ __PACKAGE__->add_unique_constraint([qw(bugid)]);
 
 # this method returns the bug if it has already been refreshed and exists (and undef otherwise)
 sub get_bug {
-    my ($self, $bugid, $db, %attrs) = @_;
+    my ($bugid, $db, %attrs) = @_;
```

Comment:
This should be a method called on a resultset, not a class method or function.

Reply:
Btw. It is convention to use `$class` instead of `$self` to describe the first argument of a class method.

Reply:
That's just the other side of that last comment :-)